tests/fs_test.py

```diff
@@ -1219,6 +1231,21 @@ def test_ext4_generic_resize(self):
                                   fs_info_func=BlockDev.fs_ext4_get_info,
                                   info_size_func=lambda fi: fi.block_size * fi.block_count)
 
+    def test_ntfs_generic_resize(self):
+        """Test generic resize function with an ntfs file system"""
+        def mkfs_prepare(drive, l):
+            return BlockDev.fs_ntfs_mkfs(drive, l) and BlockDev.fs_repair(drive)
+        def info_prepare(drive):
+            return BlockDev.fs_repair(drive) and BlockDev.fs_ntfs_get_info(drive)
+        def expected_size(fi):
+            # man ntfsresize says "The filesystem size is set to be at least one
+            # sector smaller" (maybe depending on alignment as well?), thus on a
+            # loop device as in this test it is 4096 bytes smaller than requested
```

Comment:
Is "loop device" ok as description or should it be called LIO device?

Reply:
It's a LIO loop device so I don't see a problem with this. :)